### PR TITLE
Updated README and added first script and output

### PR DESCRIPTION
**Pull request by antoniascheidel**
We also have a gitignore and a requirements file now.
We should update the requirements file whenever we
install something! I would like to write a script
that automatically does this.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+led python code files and OS/editor temp files
+*.DS_Store
+*.pyc
+.*.swp
+*~
+
+# python virtualenv
+/.pyenv/
```
Comment (lownesc) [inline .gitignore:8]: :)

File: common/util.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""
+Misc. utility functions! Mostly logging right now.
+"""
+
+LOG_COLORS = {
```
Comment (lownesc) [inline common/util.py:9]: We should eventually make an enum class.

**Comment by lownesc** [comment]
LGTM!

**Comment by lownesc** [comment]
I forget, why are the `__init__.py` files needed?

File: common/util.py (hunk 2 above)
Comment (antoniascheidel) [inline common/util.py:9]: Yeah! Adding a todo.

**Comment by lownesc** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/antoniascheidel/meaningful-lobster/pull-request/1/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=antoniascheidel/meaningful-lobster)

**Pull request merged**